packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -264,6 +265,14 @@ class SemanticsProperties extends DiagnosticableTree {
     this.onMoveCursorBackwardByCharacter,
   });
 
+  /// If non-null, indicates that this subtree represents something that can be
+  /// in a disabled or enabled state.
+  ///
+  /// For example, a button that a user can currently interact with is
+  /// considered enabled. A button that currently does not respond to user
+  /// interactions is disabled.
```

Comment:
nit: I'd replace "is considered enabled" with "would set this field to true", and "is disabled" with "would set this field to false".

Reply:
Done.